src/electron/emain.ts

```diff
@@ -485,6 +485,30 @@ electron.ipcMain.on("reload-window", (event) => {
     return;
 });
 
+electron.ipcMain.on("get-last-logs", async (event, numberOfLines) => {
+    try {
+        const logPath = path.join(getWaveHomeDir(), "wavesrv.log");
+        const lastLines = await readLastLinesOfFile(logPath, numberOfLines);
+        event.reply("last-logs", lastLines);
+    } catch (err) {
+        console.error("Error reading log file:", err);
+        event.reply("last-logs", "Error reading log file.");
+    }
+});
+
+function readLastLinesOfFile(filePath, lineCount) {
```

Comment:
this is going to be a problem if the log file becomes very large as we don't want to read it all into memory.  reading exactly the "n" last lines of a file is a hard problem.  in this case it might be easier to just run `tail -n [lines] wavesrv.log` and capture the output.

Reply:
Addressed